proxy-manager/rootfs/etc/cont-init.d/mysql.sh

```diff
@@ -22,6 +22,19 @@ password=$(bashio::services "mysql" "password")
 port=$(bashio::services "mysql" "port")
 username=$(bashio::services "mysql" "username")
 
+#Drop database based on config flag
+if bashio::config.true 'reset_database'; then
+    bashio::log.warning 'Recreating database'
+    echo "DROP DATABASE IF EXISTS nginxproxymanager;" \
+    | mysql -h "${host}" -P "${port}" -u "${username}" -p"${password}"
+    
+    #Reset addon options to null
+    curl -s -H 'X-Supervisor-Token: '"$SUPERVISOR_TOKEN"'' \
+        -H 'Content-Type: application/json'  \
+            http://supervisor/addons/self/options \
+	    --data-binary '{"options": {}}'
```

Comment:
```suggestion
    bashio::api.supervisor POST "/addons/self/options" '{"options": {}}'
```

Reply:
Guess I missed that method...

Reply:
Yes, probably better to merge the PR for bashio you made, however, that is another cycle. This reduceses it at least a bit for now.

Reply:
The bashio PR needs some more work, as from what I remember everything will be a string in the state it is in at the moment.  Updated to existing method.